suite-native/module-passphrase/src/screens/PassphraseFormScreen.tsx

```diff
@@ -1,17 +1,97 @@
+import Animated, { useSharedValue, useAnimatedStyle, withTiming } from 'react-native-reanimated';
+import { View } from 'react-native';
+
 import { Screen } from '@suite-native/navigation';
-import { Text } from '@suite-native/atoms';
+import { Box, Button, HStack, Text, VStack } from '@suite-native/atoms';
 import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Icon } from '@suite-common/icons';
 
 import { PassphraseForm } from '../components/PassphraseForm';
 import { PassphraseScreenHeader } from '../components/PassphraseScreenHeader';
 
+const ANIMATION_DURATION = 300;
+const ALERT_CARD_HEIGHT = 148;
+
+const cardStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundAlertBlueSubtleOnElevation1,
+    borderRadius: utils.borders.radii.medium,
+    padding: 12,
+    flex: 1,
+    justifyContent: 'center',
+}));
+
+const animationWrapperStyle = prepareNativeStyle(() => ({
+    overflow: 'hidden',
+}));
+
 export const PassphraseFormScreen = () => {
+    const { applyStyle, utils } = useNativeStyles();
+
+    const cardHeight = useSharedValue({ height: ALERT_CARD_HEIGHT });
```

Comment:
why is this value an object? Could not it be a number instead?
```suggestion
    const cardHeight = useSharedValue(ALERT_CARD_HEIGHT);
```

Reply:
Fixed